cocos/core/math/quat.ts

```diff
@@ -397,8 +397,12 @@ export class Quat extends ValueType {
     }
 
     /**
-     * @en Normalize the given quaternion
-     * @zh 归一化四元数
+     * @zh 归一化指定四元数。
+     * @en Normalizes the specified quaternion.
+     * @out @zh 结果四元数。@en Result quaternion.
+     * @param a @zh 输入四元数。@en Input quaternion.
+     * @note @zh 如果输入四元数 **精确** 等于零四元数，结果将设为单位四元数。
+     * @en If the input quaternion is **EXACTLY** zero vector, the quaternion will be set to identity quaternion.
```

Comment:
I think it is better to describe what is `identity quaternion`. I am not sure if it is a well known words.